### PR TITLE
Add simple test for Params

### PR DESCRIPTION
**Pull request by Cabalist**
Demo of issue referenced in #785

### CODE REVIEW

File: tests/test_django.py [hunk 1]
```diff
@@ -232,6 +243,17 @@ def test_multicall(self):
             ["alt", "main"],
         )
 
+    def test_params_blank(self):
+        obj = MultifieldModelFactoryWithParams(slug="main")
+        self.assertEqual(obj.slig, "main")
```
Comment (pokidovea) [inline tests/test_django.py:248]: typo

Comment (pokidovea) [inline tests/test_django.py:248]:
```suggestion
        self.assertEqual(obj.slug, "main")
```

Comment (Cabalist) [reply tests/test_django.py]: Thank you!  Fixed.  D'oh.

**Comment by rbarrois** [comment]
Thanks for the reproducing code, it was quite helpful in bringing a proper fix through #828 — now merged!

**Comment by Cabalist** [comment]
@rbarrois Wonderful!  Thank you!

**Pull request closed without merging**